.github/workflows/ci.yml

```diff
@@ -509,6 +509,7 @@ jobs:
           #   * various variable quoting patterns
           #   * possible ineffective echo commands
           flags: "-ignore SC2129 -ignore SC2086 -ignore SC2046 -ignore 2116"
+          files: ".github/workflows/*.yml, .github/actions/*.yml"
```

Comment:
you may need to see if it supports `**` glob patterns? Unfortunately all the actions we have are nested in another folder (i.e. `.github/actions/<action name>/action.yml`). The workflow output for the updated version of this [shows](https://github.com/alcionai/corso/actions/runs/5969083896/job/16194249377?pr=4105#step:3:372) that it still only found workflow files

Reply:
Yeah, it must use that.

Reply:
huh well, seems like it accepts `**` globs. Though now there's a bunch of errors 😭